tutorials/experiments/langchain_email_extraction.ipynb

```diff
@@ -1,269 +1,2445 @@
 {
```

Comment:
<p>A link to this metric or a high-level description would be helpful here. I'm not personally familiar with this one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/3805/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>